tsconfig.json.bac

```diff
@@ -0,0 +1,29 @@
+{
```

Comment:
Is this file doing anything? I have never seen a `.bac` file...

Reply:
oops, accidental backup file to check some settings. Will remove it.